include/vara/Feature/Feature.h

```diff
@@ -27,36 +27,53 @@ class Feature {
   using relationship_iterator = RelationshipListTy ::iterator;
 
 private:
-  string Name;
+  std::string Name;
+  llvm::SetVector<llvm::Value *> Values;
+  size_t ID;
```

Comment:
I think we can remove these two but this also requires changes on the VaRA side.